module/product/src/Application/Model/Product/Attribute/Update/UpdateAttributeValueFormModel.php

```diff
@@ -10,11 +10,7 @@
 
 use Symfony\Component\Validator\Constraints as Assert;
 use Ergonode\Attribute\Application\Validator as AttributeAssert;
-use Ergonode\Product\Application\Validator as ProductAssert;
 
-/**
- * @ProductAssert\ProductAttribute()
- */
 class UpdateAttributeValueFormModel
```

Comment:
The form no logger needs validation?

Reply:
This validation was moved to higher level as need ProductId 
https://github.com/ergonode/backend/pull/1836/files#diff-a1b027ce42013e13a0c4676d7ddce076b00255f9a3adeeb921539f07b0dec0d3R15